### DISCUSSION TITLE
A tutorial on how to use Mobi

### DISCUSSION THREAD

**Discussion by Mwilaputa** [Open-Systems-Pharmacology/Forum#1140] [Questions & Problems] [answered]
Are there any tutorials on how to extend a PBPK model from PK-Sim to Mobi?  I have tried to teach myself but failing. Is there any case example of how to model in Mobi?

**Comment by StephanSchaller** [comment, marked as answer]
We do have free tutorials for academics: https://learn-esqlabs.talentlms.com/
For industry, a cost-based solution will launch soon.

There might also be some older material here on github you could try: https://github.com/Open-Systems-Pharmacology/ISOP-webinar---PK-Sim-MoBi-introduction
